src/script/browser-script.ts

```diff
@@ -0,0 +1,3 @@
+const dummy = 'This is a placeholder for the content of built browser script.'
```

Comment:
We are using a strange trick for bundling precompiled IIFE script as string by rollup, without any errors.
https://github.com/marp-team/marp-core/pull/115/files#diff-ff6e5f22a9c7e66987b19c0199636480R23